### PR TITLE
Add SPDX header

### PR DESCRIPTION
**Pull request by AndyScherzinger**
* reuse check
* reuse badge

all compiled js files have been updated, so I guess that is correct given the license file generation and added headers in the source files, but can't tell. Looks good from a reuse perspective but can't tell more than that.

### CODE REVIEW

File: .github/workflows/lint-eslint-when-unrelated.yml
Comment (nickvergessen) [inline .github/workflows/lint-eslint-when-unrelated.yml:1]: all *-when-unrelated.yml files are invalid and should be removed.
We fixed the normal actions in the meantime

Comment (AndyScherzinger) [reply .github/workflows/lint-eslint-when-unrelated.yml:1]: Sounds good to me, while I would leave that to @artonge and @Altahrim 

But good to know, I seen them quite often when adding SPDX headers to repos

File: package.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
   "license": "AGPL-3.0-or-later",
   "scripts": {
     "build": "NODE_ENV=production webpack --progress --config webpack.js",
+	"postbuild": "build-js/npm-post-build.sh",
```
Comment (artonge) [inline package.json:10]: Weird indentation

**Review by artonge — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (AndyScherzinger) [reply package.json]: Fixed via https://github.com/nextcloud/end_to_end_encryption/pull/787/commits/a2704ae1289e6c02ffaa0470c5867347250f52e8

**Pull request merged**